README.md

```diff
@@ -1,12 +1,10 @@
-# LBRY Android
+# LBRY F-Droid
 [![pipeline status](https://ci.lbry.tech/lbry/lbry-android/badges/master/pipeline.svg)](https://ci.lbry.tech/lbry/lbry-android/commits/master)
 [![GitHub license](https://img.shields.io/github/license/lbryio/lbry-android)](https://github.com/lbryio/lbry-android/blob/master/LICENSE)
 
 An Android browser and wallet for the [LBRY](https://lbry.com) network.
 
-
-<img src="https://spee.ch/@lbry:3f/android-08-homepage.gif" alt="LBRY Android GIF" width="384px" />
-
```

Comment:
Revert this change, rebase your commit and push to your branch again.

I see the GIF is no longer loading, but this is also affecting lbry-android so it should also be fixed on that repository and then merged here.

Reply:
Okay copy that